### ISSUE TITLE
KO if no "*/10" in crontab

### ISSUE THREAD

**Issue by pibouill** [gemartin99/Born2beroot-Tester#3]
The tester gives a KO if it doesn't find a "*/10" unary operation in the crontab 

There are other ways to make the script run without using this method 

(My bad if this is an actual mandatory requirement)

**Comment by gemartin99** [comment]
The checker gives a KO if it does not find a "*/10" unary operation in the crontab.

There are other ways to make the script run without using this method.

(My bad if this is a mandatory requirement).

You are right , it can be executed in other ways, now I am focused on other things, but thanks for commenting , I will try to change it in the future and put OK in all cases where it can be put every 10 minutes. Thanks for improving the tester

Translated with www.DeepL.com/Translator (free version)

**Comment by gemartin99** [comment]
You are right , it can be executed in other ways, now I am focused on other things, but thanks for commenting , I will try to change it in the future and put OK in all cases where it can be put every 10 minutes. Thanks for improving the tester
